marketredesign/mrd-auth0-laravel
# Custom NewRelic logger

## Proposed changes
Adds a custom logger for logging to NewRelic, including various default fields.

Related to [AB#42](https://dev.azure.com/jdokter/973dd796-5b61-4e4a-a52b-ab66f88cef8a/_workitems/edit/42)

## Types of changes

## Testing
### Unit testing
None specifically for this.

### Manual testing
Use the logger in a repository (see README.md on how to activate it) and ensure it works and is robust.

## Further comments
